CHANGELOG.md

```diff
@@ -1,4 +1,21 @@
 # Changelog
+## [6.0.0] - 2022-10-04
+- Added the ability to specify simple fields in a more compact way, inline.
+
+Example: id and id2, name and name2 are equivalent notations
+```yaml
+ExampleModel:
+  properties:
+    id: string
+    id2: 
+      type: string
```

Comment:
Maybe we should move our documentation & changelog to the correct types? `String` and `String?` instead of `string` and `string?`

Reply:
Given that both variants are supported 🤷 Will update